en/docs/deploy/secure-userstore-using-pbkdf2.md

```diff
@@ -0,0 +1,75 @@
+# PBKDF2 hashing
+We can use Password-Based Key Derivation Function 2 (PBKDF2) hashing method to securely store user passwords in user stores. This method reduces the risk of brute-force attacks due to insecure passwords.
```

Comment:
better to have a sentence about PBKDF2 being a modern password hashing algorithm backed by NIST standards.

Reply:
+1